nf_core/pipelines/sync.py

```diff
@@ -410,74 +409,6 @@ def make_pull_request(self):
                 log.debug(f"GitHub API PR worked, return code {r.status_code}")
                 log.info(f"GitHub PR created: {self.gh_pr_returned_data['html_url']}")
 
-    def close_open_template_merge_prs(self):
```

Comment:
I wouldn't remove this functions, maybe we want to reimplement this in the future? But we could deprecate them for now, maybe add a TODO comment or something similar?

Reply:
yes, better to not have unused functions in the code. If we want to redo this, we hopefully remember what we had 😁

Reply:
ok! I approved then